### ISSUE TITLE
Create audit rubric

### ISSUE THREAD

**Issue by kellysmith1008** [usds/ditap-curriculum-update#54]
**Dependencies**
- Alignment on approach (notes from audit kickoff call on 10/16)-Access to notes document

**Definition of Success**
- We have created an audit rubric to be used by the team in the audit process.
- Rubric contains clearly defined criteria and instructions for use.
- Rubric has been reviewed by all team members.
- Team understands how to use rubric.

**Other information**
- See meeting notes link for notes from audit kickoff https://docs.google.com/document/d/1RZHovtejf5f780lItuzOb9IIHIAWVXKFQk1EqafvDLo/edit?tab=t.0

**Comment by kellysmith1008** [comment]
We've created [a first pass at the audit rubric](https://docs.google.com/spreadsheets/d/15ryjqGEn9NCMztE0RjSYvNHlwWxCiU3MPUL3rCr3L3g/edit?usp=sharing) and shared in in the slack channel. We've also created a folder for team members to place their completed rubrics when we begin #58 . 

After team members have completed their rubrics, we will create another ticket to synthesize the rubric data and share out with the team. I'll update the ticket number here when it's created.

**Comment by kellysmith1008** [comment]
Please provide any feedback by noon October 23, 2024

**Comment by Chianti7** [comment]
Reviewed and added comments in spreadsheet.

**Comment by TahiraG** [comment]
Added comments directly in doc.

**Comment by cruzaprild** [comment]
@kellysmith1008 I'm sorry that I didn't surface this before this ticket made it to done. If we haven't already, can we get Mike's feedback on the Accessibility section of this before the sprint ends tomorrow? It would be great to capture any other feedback he may have on that tab before we start working through audit tickets in a couple of days. cc: @mgifford
